lib/App/index.js

```diff
@@ -285,10 +285,29 @@ class App {
           await this._validateImages(driver.images, 'driver');
         }
 
+        // validate `appJson.drivers[].connectivity`
```

Comment:
This comment seems to be too high

Reply:
Duplicated some of the checks so that the code is clearer and the comments make more sense 👍